api/tests/Feature/PoolCandidateSearchRequestTest.php

```diff
@@ -344,4 +344,80 @@ public function testUserIdForLoggedInUsers()
 
         $this->assertTrue(PoolCandidateSearchRequest::where('user_id', $this->adminUser->id)->exists());
     }
+
+    public function testUserCanSeeTheirOwnRequests()
+    {
+        $this->seed(DepartmentSeeder::class);
+        $this->seed(CommunitySeeder::class);
+
+        $user1 = User::factory()->create();
+        $request1 = PoolCandidateSearchRequest::factory()->create(['user_id' => $user1->id]);
+        $user2 = User::factory()->create();
+        $request2 = PoolCandidateSearchRequest::factory()->create(['user_id' => $user2->id]);
```

Comment:
Might be overkill, but can we create a third request with no user attached (user_id = "null") to ensure they dont show up as well?